src/cbmc/bmc.h

```diff
@@ -71,6 +71,20 @@ class bmct:public safety_checkert
     return run(goto_functions);
   }
 
+  /// Retrieves the bmc equation
+  /// @return a reference to the bmc equation object
+  const symex_target_equationt& get_equation() const
+  {
+    return equation;
+  }
+
+  /// Retrieves the prop_conv field
+  /// @return a reference to the prop_conv member
+  const prop_convt & get_prop_conv() const
```

Comment:
Please use consistent spacing around `&` - see also clang-format's output: it should be `prop_convt &get_prop_conv`. Fix `get_equation` above accordingly.